src/Controllers/ImageController.php

```diff
@@ -72,6 +72,29 @@ public function __invoke(Request $request, int $store, string $size, string $pla
         return $this->storage()->response($resizedPath);
     }
 
+    public function redirectFromSku(Request $request, int $store, string $size, string $file)
+    {
+        $webp = str_ends_with($file, '.webp') ? '.webp' : '';
+        $sku = $webp ? str_replace_last('.webp', '', $file) : $file;
+        $file = $this->productImageUrlFromSku($sku);
+        $placeholder = 'magento';
+
+        return redirect(route('resized-image', @compact('store', 'size', 'placeholder', 'file', 'webp')), 301)->setPublic()->setMaxAge(3600);
```

Comment:
Maybe better a 302 as the image _could_ change? And maybe make it configurable?
```
// Enable image URL's by SKU like:
// /storage/1/resizes/80x70/SKU
'sku' => [
    'enabled' => true,
    'status_code' => 302,
    'max_age' => 86400, // One day
],
```

Reply:
Being able to configure it is definitely essential, I agree.

The status code could also be 307, but I guess 302 is more common and thus a safer bet?